src/jobs/schemas/job.schema.ts

```diff
@@ -121,7 +131,45 @@ export class JobClass extends OwnableClass {
   })
   jobParams: Record<string, unknown>;
 
-  // TBD email address for owner from scicat? see create example for job.recipients
+  // dataset validation results
+  @ApiProperty({
+    type: Boolean,
+    required: true,
+    description:
+      "True if the user has the access requested to all the datasets listed on this job",
+  })
+  @Prop({
+    type: Boolean,
+    required: true,
+  })
+  datasetsValidation: boolean;
+
+  @ApiProperty({
+    type: String,
+    required: false,
+    default: "",
+    description:
+      "Email of the person to contact regarding this job. If the job is submitted anonymously, an email has to be provided",
+  })
+  @Prop({
+    type: String,
+    required: false,
+  })
+  contactEmail: string;
+
+  @ApiProperty({
+    type: String,
+    required: true,
+    default: {},
+    description: "Configuration that was used to create this job.",
+  })
+  @Prop({
+    type: Object,
+    required: true,
+  })
+  configuration: Record<string, unknown>;
```

Comment:
I believed we had agreed to store only the config version instead of the full configuration, to avoid overloading the job schema

Reply:
I think this one it's better to discuss in a meeting. When we read to job config file we store its contents in the configuration so that it's easily accessible from there. However, I understand the pros and cons of both cases, so let's talk about it again

Reply:
we will explore different way to write this